fuel-coroutines/src/test/kotlin/com/github/kittinunf/fuel/coroutines/CoroutinesTest.kt

```diff
@@ -39,8 +38,8 @@ class CoroutinesTest {
     @Test
     fun testAwaitResponseSuccess() = runBlocking {
         mock.chain(
-            request = mock.request().withPath("/ip"),
-            response = mock.reflect()
+                request = mock.request().withPath("/ip"),
```

Comment:
[Nits] Not sure about these indentations though? Have you tried to use the auto format from Intellij/AS?

Reply:
that is what I am using

Reply:
The auto format isn't completely sane by default. It will "over indent" function parameters on a new line. Perhaps one of us should toy with the settings, and then make the settings available (you can export/import them).